tests/tests_geomstats/test_normal.py

```diff
@@ -103,21 +103,21 @@ def test_unstack_mean_covariance(
         self.assertAllClose(mean.shape, mean_expected)
         self.assertAllClose(cov.shape, cov_expected)
 
-    @pytest.mark.skip(reason="Flaky test.")
+    @pytest.mark.xfail
```

Comment:
I'm a little bit surprised that all these tests - even for simple functions such as belongs - fail sometimes for normal distributions. Do you have any idea why ?

Reply:
These tests were skipped by @johmathe in the end of the last year. I'm not sure why, but I'm guessing they were failing once in a while. I've just updated them to be `xfail`, so we run them anyway, but I haven't really checked how often they fail.